lbp/src/components/Wallet/AccountStatus.tsx

```diff
@@ -0,0 +1,392 @@
+        justify-content: center;
+        padding: 14px 16px;
+        width: ${props.showInvestButton ? `${walletButtonWidth}%` : "90%"};
+      `;
+      case "desktop":
+        return ``;
+    }
+  }}
+`;
+
+const WalletButtonText = styled(Title)<WalletStatusProps>`
+  font-size: 14px;
+  line-height: 20px;
+
+  @media (max-width: ${sizes.lg}px) {
+    font-size: 16px;
+  }
+
+  @media (max-width: 350px) {
+    font-size: 13px;
+  }
+
+  ${(props) => {
+    if (props.connected) return null;
+
+    return `color: ${colors.green}`;
+  }}
+`;
+
+const WalletDesktopMenu = styled.div<MenuStateProps>`
+  ${(props) =>
+    props.isMenuOpen
+      ? `
+          position: absolute;
+          right: 40px;
+          top: 64px;
+          width: fit-content;
+          background-color: ${colors.backgroundDarker};
+          border-radius: ${theme.border.radius};
+        `
+      : `
+          display: none;
+        `}
+
+  @media (max-width: ${sizes.lg}px) {
+    display: none;
+  }
+`;
+
+const WalletMobileOverlayMenu = styled(
+  MobileOverlayMenu
+)<AccountStatusVariantProps>`
+  display: none;
+
+  ${(props) => {
+    switch (props.variant) {
+      case "mobile":
+        return `
+          @media (max-width: ${sizes.lg}px) {
+            display: flex;
+            z-index: ${props.isMenuOpen ? 50 : -1};
+          }
+        `;
+      case "desktop":
+        return ``;
+    }
+  }}
+`;
+
+const MenuItem = styled.div`
+  padding: 8px 16px;
+  padding-right: 38px;
+  opacity: 1;
+  display: flex;
+  align-items: center;
+
+  &:first-child {
+    padding-top: 16px;
+  }
+
+  &:last-child {
+    padding-bottom: 16px;
+  }
+
+  &:hover {
+    span {
+      color: ${colors.primaryText};
+    }
+  }
+
+  @media (max-width: ${sizes.lg}px) {
+    margin: unset;
+    && {
+      padding: 28px;
+    }
+  }
+`;
+
+const MenuItemText = styled(Title)`
+  color: ${colors.primaryText}A3;
+  white-space: nowrap;
+  font-size: 14px;
+  line-height: 20px;
+
+  @media (max-width: ${sizes.lg}px) {
+    font-size: 24px;
+  }
+`;
+
+const WalletCopyIcon = styled.i<WalletCopyIconProps>`
+  color: white;
+  margin-left: 8px;
+  transition: 0.1s all ease-out;
+
+  ${(props) => {
+    switch (props.state) {
+      case "visible":
+        return `
+          opacity: 1;
+        `;
+      case "hiding":
+        return `
+          opacity: 0;
+        `;
+      case "hidden":
+        return `
+          visibility: hidden;
+          height: 0;
+          width: 0;
+          opacity: 0;
+        `;
+    }
+  }}
+`;
+
+const MenuCloseItem = styled(MenuItem)`
+  position: absolute;
+  bottom: 50px;
+  left: 0;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+`;
+
+interface AccountStatusProps {
+  variant: "desktop" | "mobile";
+}
+
+const truncateAddress = (address: string) => {
+  return address.slice(0, 6) + "..." + address.slice(address.length - 4);
+};
+
+const AccountStatus: React.FC<AccountStatusProps> = ({ variant }) => {
```

Comment:
Why are we copy-pasting this component into `lbp` instead of reusing it from `shared`?

Reply:
There is actually few key differences between Account Status in LBP with Account Status in Webapp, mainly Webapp Account Status include invest button which LBP does not share with. Account status is also a wrapper for `ConnectWalletButton` which both shared, so I make a concious decision to separate them to their own repo to keep the distinction apart.